agent/prototyper.py

```diff
@@ -129,14 +128,14 @@ def _container_handle_conclusion(
       self.logger.info('***** Prototyper succeded in %d rounds *****',
                        cur_round,
                        extra={'trial': self.trial})
-      self.write_to_file(
-          os.path.join(build_result.work_dirs.fixed_targets,
-                       f'{build_result.trial}.fuzz_target'),
-          build_result.fuzz_target_source)
-      self.write_to_file(
-          os.path.join(build_result.work_dirs.fixed_targets,
-                       f'{build_result.trial}.build_script'),
-          build_result.build_script_source)
+      # self.write_to_file(
+      #     os.path.join(build_result.work_dirs.fixed_targets,
+      #                  f'{build_result.trial}.fuzz_target'),
+      #     build_result.fuzz_target_source)
+      # self.write_to_file(
+      #     os.path.join(build_result.work_dirs.fixed_targets,
+      #                  f'{build_result.trial}.build_script'),
+      #     build_result.build_script_source)
```

Comment:
Currently `write_to_file` only makes sense for local experiments, will address this in a better way soon.

Reply:
I really wish we had a library like [TF's gfile](https://www.tensorflow.org/api_docs/python/tf/io/gfile/GFile). Then we could use the same code path to write either locally or on GCP, etc. depending on how we run things.

But right now this requires importing the full machinery of TF :(